### DISCUSSION TITLE
Magic Link besides admin is not getting send

### DISCUSSION THREAD

**Discussion by alfainfotech** [mozilla/hubs#4930] [Hubs Cloud Questions] [answered]
Hello,

anyone here has issue with magic link not been send to new users. As admin I get the magic link via SES default, but other users are not getting and followed other threads solutions even increased EC2 size, and never used any SMTP etc. 

Any help is appreciated

**Comment by tranvu14** [comment]
Me too :((  I'm facing this.

**Comment by pattersonbl2** [comment]
Have you gone into the SES console in AWS and see if you have any errors ?

**Reply by alfainfotech** [reply]
Yes no error I checked SES console. If I send manually from SES console as test it goes fine.

**Reply by pattersonbl2** [reply]
This point to a issue with the reticulum config.  you would probably need to ssh into Ec2 and check the smtp setting.

**Reply by alfainfotech** [reply]
Thanks for your help. I did manual SMTP setup on admin, now I even lost the admin email link when I sign up. Delete all the instances/stacks and re-depolying on AWS. let see if this do anything.

**Comment by alfainfotech** [comment]
Re-installed few times, now I'm not even getting Admin email link to login ? Anyone in same situation.

**Comment by alfainfotech** [comment]
In my Stacks I see this, its been few hrs still it status saying 'in..progress..... Any idea what to do

SendEmailPasswordSecret | CREATE_IN_PROGRESS | Resource creation Initiated

**Comment by alfainfotech** [comment, marked as answer]
Okay after doing it from fresh re-install things are working and getting emails.
